src/index.js

```diff
@@ -20,6 +21,7 @@ export default class ReactPasswordStrength extends React.Component {
       score: 0,
       isValid: false,
       password: '',
+      feedback: {}
```

Comment:
would prefer that we dont keep `feedback` in the state just for the reason of passing it along in the callback on ln 62

Reply:
Makes sense!